oracle/datadog_checks/oracle/oracle.py

```diff
@@ -80,10 +79,8 @@ def __init__(self, name, init_config, instances):
             tags=self._tags,
         )
 
```

Comment:
Since Python 2 is no longer supported for the Oracle integration, can we add a [condition](https://github.com/DataDog/integrations-core/blob/master/argocd/datadog_checks/argocd/check.py#L25-L30) that checks using Python 2, and raise a `ConfigurationError`?

Reply:
Done